package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "f3d",
+  "version": "3.2.0",
+  "description": "Fast and minimalist 3D viewer framework",
+  "exports": "./dist/f3d.js",
+  "files": [
+    "./dist/f3d.*"
+  ],
+  "scripts": {
+    "_docker": "${F3D_DOCKER_CLIENT:-docker} run --rm -e CMAKE_BUILD_PARALLEL_LEVEL -v $(pwd):/src ghcr.io/f3d-app/f3d-wasm:${F3D_DOCKER_TIMESTAMP:-latest}",
```

Comment:
so `F3D_DOCKER_TIMESTAMP` must be set in the env of the user ?

Thats not great imo. Can you explain what this does and how you are using it ?

Reply:
Not really. I will document it tho